### PR TITLE
Put up Blog #2 - Coding Academy Survival Guide

### PR DESCRIPTION
**Pull request by LuckyHariharan**
(https://docs.google.com/document/d/1S_hWJ1NgjRGNb1HyzE0STa25F-8w5We8oviVl62C4UY/edit)
image

In this PR i put up my coding academy survival guide with 18 curated tips. for some reason im struggling to get the numbers from the numbered-list li tag to be on the same row as the text, it seems to only appear above the text. this is why im not using a numnered list in the JSX

### CODE REVIEW

**Review by vithushan19 — CHANGES_REQUESTED** [review]
can you add your blog posts to sitemap.xml as well, so that google can find them.

File: public/sitemap.xml [hunk 1]
```diff
@@ -15,6 +15,11 @@
         <lastmod>2022-05-17T23:23:47.440Z</lastmod>
         <changefreq>daily</changefreq>
     </url>
+    <url>
+        <loc>https://skillify.ca/blog/coding-academy-tips</loc>
+        <lastmod>2022-05-19T23:23:47.440Z</lastmod>
+        <changefreq>daily</changefreq>
+    </url>
     <url>
```
Comment (LuckyHariharan) [inline public/sitemap.xml:23]: updated sitemap for google

**Comment by LuckyHariharan** [comment]
> can you add your blog posts to sitemap.xml as well, so that google can find them.

@vithushan19 just pushed the sitemap change, ready for review

File: pages/blog/coding-academy-tips.tsx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        </div>
+        <div className="mt-4 FLEX">
+          <div>
+            <div className="font-bold">Collaborate with others.</div> Working
+            with classmates or colleagues on projects can help you learn from
+            their strengths and weaknesses, while also building your teamwork
+            skills.
+          </div>
+        </div>
+        <div className="mt-4 FLEX">
```
Comment (vithushan19) [inline pages/blog/coding-academy-tips.tsx:134]: why do all of these have FLEX capitalized? do you even need these to be flex?

**Review by vithushan19 — APPROVED** [review]
This is a great post, nice work. I think this will help explain to people what you can learn from Skillify. I think you can also add your name and linkedin profile if you think it'll help in marketing yourself as a strong/communicative developer.

**Comment by LuckyHariharan** [comment]
> This is a great post, nice work. I think this will help explain to people what you can learn from Skillify. I think you can also add your name and linkedin profile if you think it'll help in marketing yourself as a strong/communicative developer.

ty for your reply, will add my details!

**Pull request merged**